PrismSampleApp/PrismSampleApp/ViewModels/ViewCViewModel.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PrismSampleApp.ViewModels
+{
+    public class ViewCViewModel : BindableBase, INavigationAware
```

Comment:
Name class properly and use ViewmodeBase

Reply:
ViewModelBase has been used here.